src/MaaCore/Task/Miscellaneous/StartGameTaskPlugin.cpp

```diff
@@ -9,7 +9,18 @@ bool StartGameTaskPlugin::_run()
     if (m_client_type.empty()) {
         return false;
     }
-    return ctrler()->start_game(m_client_type);
+
+    do {
+        if (!ctrler()->start_game(m_client_type)) {
+            return false;
+        }
+
+        if (ctrler()->get_pipe_data_size() > 167) {
```

Comment:
Is 167 a magic number?

Reply:
Yes.
I forgot to transfer the comment from the closed PR.
I'll do it later.